src/benchmarks/gc/GC.Infrastructure/GC.Analysis.API/Analyzer.cs

```diff
@@ -38,7 +38,7 @@ public Analyzer(string tracePath, HashSet<string> processNames)
                     AllGCProcessData[p.Name] = values = new();
                 }
 
-                values.Add(new GCProcessData(p, managedProcess, processIdToGCThreads[p.ProcessID], this, p.EndTimeRelativeMsec - p.StartTimeRelativeMsec));
+                values.Add(new GCProcessData(p, managedProcess, processIdToGCThreads.GetValueOrDefault(p.ProcessID) ?? new(), this, p.EndTimeRelativeMsec - p.StartTimeRelativeMsec));
```

Comment:
@mrsharm This (and the similar change below) are new.  I was trying to analyze a trace that didn't have this mapping.  (If I recall correctly, I have a notebook side change to not filter these out, but this access will throw without this change.)

Reply:
Looks good!